tags.go

```diff
@@ -30,3 +33,22 @@ func GetTagValue(tags []string, key string, separator string) string {
 
 	return ""
 }
+
+// AddNewTag adds a new tag at the beginning of the comma-separated tag list
```

Comment:
```golang
// Add adds a new tag to the comma-separated tag list
func (tags Tags) Add(tag string) Tags {
	return Tags(strings.Join(append(tags.Split(), tag), ","))
}
```

Would call it add also because it reads better `tags.Add`